configs/envoy_front_proxy_v2.template.yaml

```diff
@@ -0,0 +1,158 @@
+          {{ router_file_content(router_file='envoy_router_v2.template.yaml')|indent(10) }}
+        http_filters:
+        - name: envoy.health_check
+          config:
+            pass_through_mode: false
+            endpoint: "/healthcheck"
+        - name: envoy.buffer
+          config:
+            max_request_bytes: 5242880
+            max_request_time: 120s
+        - name: envoy.rate_limit
+          config:
+            domain: envoy_front
+            request_type: external
+        - name: envoy.router
+          config: {}
+        add_user_agent: true
+        tracing:
+          operation_name: INGRESS
+        idle_timeout: 840s
+        access_log:
+        - name: envoy.file_access_log
+          filter:
+            or_filter:
+              filters:
+                - status_code_filter:
+                    comparison:
+                      op: GE
+                      value:
+                        default_value: 500
+                        runtime_key: access_log.access_error.status
+                - duration_filter:
+                    comparison:
+                      op: GE
+                      value:
+                        default_value: 1000
+                        runtime_key: access_log.access_error.duration
+                - traceable_filter: {}
+          config:
+            path: /var/log/envoy/access_error.log
+            format: "[%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%REQ(X-LYFT-USER-ID)%\" \"%RESP(GRPC-STATUS)%\"\n"
+{% endmacro -%}
+static_resources:
+  listeners:
+    # TCP listeners for public HTTP/HTTPS endpoints. Assumes a TCP LB in front such as ELB which
+    # supports proxy proto.
+  - {{ listener("TCP", "0.0.0.0", "9300", True, True)|indent(2) }}
+  - {{ listener("TCP", "0.0.0.0", "9301", True, True)|indent(2) }}
+    # TCP listener for backhaul traffic from the double proxy.
+    # See envoy_double_proxy.template.json
+  - {{ listener("TCP", "0.0.0.0", "9400", True, True, pin_double_proxy_client=True)|indent(2) }}
+  clusters:
+  - name: sds
+    type: STRICT_DNS
+    connect_timeout: 0.25s
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+       protocol: TCP
+       address: disccovery.yourcompany.net
+       port_value: 80
+  - name: statsd
+    type: STATIC
+    connect_timeout: 0.25s
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+        protocol: TCP
+        address: 127.0.0.1
+        port_value: 8125
+  - name: lightstep_saas
+    type: LOGICAL_DNS
+    connect_timeout: 1s
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+        protocol: TCP
+        address: collector-grpc.lightstep.com
+        port_value: 443
+    http2_protocol_options: {}
+  - name: main_website
```

Comment:
I still think that collapsing the numbered `service` clusters from before removes semantic diversity in the config. Any strong reason you have for having collapsed this?

Reply:
overlooked it, will fix. Thanks